plugins/nextcloud/js/hideInhouseAddressbook.js

```diff
@@ -0,0 +1,45 @@
+(rl => {
+	if (rl) {
+		addEventListener('rl-view-model', e => {
+			if ('MailFolderList' === e.detail.viewModelTemplateID) {
+				const container = e.detail.viewModelDom.querySelector('.buttonContacts');
+				if (container) {
+					container.remove();
+				}
+			}
+		});
+
+
+		addEventListener('rl-view-model', e => {
+			if ('SystemDropDown' === e.detail.viewModelTemplateID) {
+				const container = e.detail.viewModelDom.querySelector('.dropdown-menu');
+				if (container) {
+					for (i = 0; i < container.children.length; i++) {
+						const element = container.children[i];
+						const attr = element.getAttribute("data-bind");
+						if (attr && attr.includes("visible: allowContacts")) {
+							element.remove();
+							break;
+						}
+					}
+				}
+			}
+		});
+
+		addEventListener('rl-view-model', e => {
+			if ('PopupsCompose' === e.detail.viewModelTemplateID) {
+				const container = e.detail.viewModelDom.querySelector('.pull-right');
+				if (container) {
```

Comment:
use  [guard clauses](https://refactoring.guru/replace-nested-conditional-with-guard-clauses) to remove nested conditionals

Reply:
@akhil1508 please check again